Slim/Plugin/InternetRadio/strings.txt

```diff
@@ -38,7 +38,7 @@ PLUGIN_RADIOTIME_MODULE_NAME_DESC
 	EN	TuneIn Radio - Local Internet Radio
 	ES	TuneIn Radio - Radio por Internet local
 	FI	TuneIn Radio - paikallinen internetradio
-	FR	TuneIn Radio - Radio Internet locale
+	FR	TuneIn Radio - Radios Internet
```

Comment:
I'm not sure this is a correct modification. This string really is about (geographically) local radio stations.

Reply:
Are you sure ?
Isn't it rather the RADIOTIME_LOCAL string which lists the geographically local radios?

PLUGIN_RADIOTIME_MODULE_NAME_DESC isn't it there to describe the plugin in the list of plugins? (Unless it's PLUGIN_INTERNET_RADIO_MODULE_NAME_DESC? I have a doubt ...)